plugins/lookup/revbitspss.py

```diff
@@ -0,0 +1,98 @@
+# -*- coding: utf-8 -*-
+# Copyright: (c) 2021, RevBits <info@revbits.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+DOCUMENTATION = r"""
+name: revbitspss
+author: RevBits (@info) <info@revbits.com>
+short_description: Get secrets from RevBits PAM server
+version_added: 1.0.1
```

Comment:
```suggestion
version_added: 3.8.0
```
The next collection release will be 3.8.0.